src/Controllers/WorkerController.php

```diff
@@ -79,7 +79,7 @@ public function queue(Request $request, WorkerInterface $worker, Container $lara
 
         $worker->process(
             $request->header('X-Aws-Sqsd-Queue'), $job, [
-                'maxTries' => 0,
+                'maxTries' => (isset($body['maxTries']) && !is_null($body['maxTries']))? $body['maxTries'] : 0,
```

Comment:
hmm, but isn't $body a string?

Reply:
Yes, true, its return string not array. need to change.

Reply:
@dusterio Fixed it.